### PR TITLE
#162383045 Add test to report article model

### PR DESCRIPTION
**Pull request by ekpangmichael**
#### What does this PR do?
Implement Unit test for sequelize models
#### Description of Task to be completed?
- Write unit test to check for relationship and properties
- Install sequelize-test-helpers and  sinon package
#### How should this be manually tested?
Clone the repo,  and run `` npm install  `` ``  npm run test``
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A

### CODE REVIEW

File: tests/models/reportArticles.test.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+const {
+    sequelize,
+    dataTypes,
+    checkModelName,
+} = sequelizeHelper;
+
+describe('## Report Articles model', () => {
+    const ReportArticleModel = ReportArticles(sequelize, dataTypes)
+    const instance = new ReportArticleModel()
+    checkModelName(ReportArticleModel)('ReportArticles')
```
Comment (V-84) [inline tests/models/reportArticles.test.js]: nice work, but you have to replace the checkmodeName() with the it() block

File: tests/models/reportArticles.test.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    context('properties tests', () => {
+        it('should have a property of report', () => {
+            expect(instance).to.have.property('report');
+        });
+    });
+    context('Report Articles associations should', () => {
+        const User = user(sequelize, dataTypes);
+        before(() => {
+            ReportArticleModel.associate({ User })
```
Comment (V-84) [inline tests/models/reportArticles.test.js]: no semicolon, please add for consistency

File: tests/models/reportArticles.test.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        })
+        it('should have a belongsTo relationship with user model', () => {
+            expect(ReportArticleModel.belongsTo.calledWith(User)).to.equal(true)
```
Comment (V-84) [inline tests/models/reportArticles.test.js]: no semi-colon, please add for consistency

**Review by efalayi — APPROVED** [review]
LGTM

**Pull request merged**